docs/examples/page-header/breadcrumb.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <el-page-header>
+    <template #breadcrumb>
+      <el-breadcrumb separator="/">
+        <el-breadcrumb-item :to="{ path: './page-header.html' }"
+          >homepage</el-breadcrumb-item
+        >
+        <el-breadcrumb-item
+          ><a href="./page-header.html">route 1</a></el-breadcrumb-item
+        >
```

Comment:
```suggestion
        <el-breadcrumb-item :to="{ path: './page-header.html' }">
          homepage
        </el-breadcrumb-item>
        <el-breadcrumb-item>
          <a href="./page-header.html">route 1</a>
        </el-breadcrumb-item>
```

Do you want to optimize the code format?

Reply:
I used `pnpm prettier` command here to format the code, I think this might related to the prettier configuration 🤔

Reply:
Formatting is sometimes not smart enough, so we need to manually modify the code format.

Reply:
Formatting is sometimes not smart enough, so we need to manually modify the code format.

Reply:
Done, PTAL :bow: